rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -43,3 +58,43 @@ impl Timer {
         LIVESPLIT_STATE.lock().unwrap().timer.set_game_time(TimeSpan::from_seconds(time));
     }
 }
+impl Run {
+    pub fn game_name() -> String {
+        LIVESPLIT_STATE.lock().unwrap().timer.run().clone().game_name().to_owned()
+    }
+    pub fn set_game_info(game: Game, category: String, new_game_glitch: NewGameGlitch) {
+        let mut state = LIVESPLIT_STATE.lock().unwrap();
+        let mut run = state.timer.run().clone();
+        let game = match game {
+            Game::Refunct => "Refunct",
+            Game::RefunctCategoryExtensions => "Refunct Category Extensions",
+            Game::RefunctMultiplayer => "Refunct Multiplayer",
+            Game::RefunctRandomizer => "Refunct Randomizer",
+        };
+        run.set_game_name(game);
+        run.set_category_name(category);
+        if run.metadata_mut().speedrun_com_variables.iter().any(|(name, value)| name == "New Game Glitch") {
+            let cat = match new_game_glitch {
+                NewGameGlitch::Yes => "New Game Glitch",
+                NewGameGlitch::No => "Normal",
+            };
+            run.metadata_mut().set_speedrun_com_variable("New Game Glitch", cat);
+        }
+        state.timer.set_run(run).unwrap();
+    }
+    pub fn category_name() -> String {
+        LIVESPLIT_STATE.lock().unwrap().timer.run().category_name().to_owned()
+    }
+    pub fn get_segments() -> Vec<Segment> {
```

Comment:
Getters in rust usually don't start with `get_*`, see [C-GETTER](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter).